src/patternfly/components/ActionList/action-list.scss

```diff
@@ -0,0 +1,28 @@
+.pf-c-action-list {
+  --pf-c-action-list--Gap: var(--pf-global--spacer--md);
+  --pf-c-action-list--m-group--Gap: var(--pf-global--spacer--2xl);
+  --pf-c-action-list--m-group--m-vertical--Gap: var(--pf-global--spacer--lg);
+  --pf-c-action-list--m-vertical--Gap: var(--pf-global--spacer--sm);
+
+  &.pf-m-group {
+    gap: var(--pf-c-action-list--m-group--Gap);
```

Comment:
This comes with the caveat that it doesn't work in safari, and we need to support the latest version of Safari. I wonder if we should build it so it works cross-browser, but with vars that will easily update once gap is supported.